nbdev/release.py

```diff
@@ -161,7 +159,6 @@ def pypi_details(name):
     "Version, URL, and SHA256 for `name` from pypi"
     ver = str(latest_pypi(name))
     pypi = pypi_json(f'{name}/{ver}')
-    info = pypi['info']
```

Comment:
Was this line of intended to do something but forgotten?

Reply:
Dunno